source/Halibut/Transport/Proxy/HttpProxyClient.cs

```diff
@@ -186,6 +186,27 @@ public IProxyClient WithTcpClientFactory(Func<TcpClient> tcpClientfactory)
         /// port.  
         /// </remarks>
         public TcpClient CreateConnection(string destinationHost, int destinationPort, TimeSpan timeout)
+        {
+            return CreateConnection(destinationHost, destinationPort, timeout);
+        }
+
+        /// <summary>
```

Comment:
Not usually a fan of xmldoc comments, but needed to emphasize what the cancellation token would be used for